recipes/iiboost/build.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export ENABLE_OPENMP=no
+else
+    export ENABLE_OPENMP=yes
+fi
+
+mkdir build && cd build
+cmake \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DITK_DIR=$PREFIX/lib/cmake/ITK-4.11/ \
+    -DIIBOOST_USE_OPENMP=$ENABLE_OPENMP \
+	..
+
+make
```

Comment:
I guess it is using `joblib` to do this. FWICT `scikit-learn` developers ran into the same problem and solved it by having separate files for Python 2 and 3.